src/__tests__/suggestions.js

```diff
@@ -0,0 +1,201 @@
+import {configure} from '../config'
+import {screen} from '..'
+import {renderIntoDocument} from './helpers/test-utils'
+
+beforeAll(() => {
+  configure({showSuggestions: true})
+})
+
+afterAll(() => {
+  configure({showSuggestions: false})
```

Comment:
"showSuggestions" is probably not the best config option since it throws. Maybe "throwSuggestions" or "failOnSuboptimalQueries" or something 🤔

Reply:
i can go w/ throwSuggestions. :)